src/main/java/wooteco/subway/admin/repository/StationRepository.java

```diff
@@ -1,9 +1,22 @@
 package wooteco.subway.admin.repository;
 
+import org.springframework.data.jdbc.repository.query.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 import wooteco.subway.admin.domain.Station;
 
-import java.util.Set;
+import java.util.List;
 
 public interface StationRepository extends CrudRepository<Station, Long> {
+    @Override
+    List<Station> findAllById(Iterable<Long> longs);
```

Comment:
```suggestion
    List<Station> findAllById(Iterable<Long> ids);
```

longs는 의미를 전달하기 적합하지 않아 보이네요~

Reply:
테스트를 분리하라고 하신 것은 다이나믹 테스트를 말씀하신 것 맞는지 궁금합니다.
아니면 단순히 `@BeforeEach` 등을 사용해서 기존에 하던 테스트를 구현하면 되는건지 궁금합니다 :)

Reply:
가독성이 떨어지거나 실패 범위가 넓어지는 것을 방지하는 관점에서 두가지 모두 가능할 것 같은데요!
지금은 그렇게 많지 않아 유지하셔도 될 것 같습니다~